client/landing/gutenboarding/hooks/use-on-signup.ts

```diff
@@ -24,6 +25,12 @@ export default function useOnSignup() {
 	const newUser = useSelect( ( select ) => select( USER_STORE ).getNewUser() );
 	const newSite = useSelect( ( select ) => select( SITE_STORE ).getNewSite() );
 	const visibility = useNewSiteVisibility();
+	const anchorFmPodcastId = useAnchorFmQueryParam();
+	const isAnchorFmSignup = useSelect( ( select ) => select( ONBOARD_STORE ).getSelectedPodcastId() );
+
+	React.useEffect( () => {
+		dispatch( ONBOARD_STORE ).setPodcastId( anchorFmPodcastId );
+	}, [] );
```

Comment:
Do we want this to run only once? What if `isAnchorFmSignup` changes?

Reply:
Filled out deps arrays here 30b0272d3c

Reply:
I had to undo the change here: 61d890df14.  The anchor sign up process no longer works after this is set to update whenever `anchorFmPodcastId` changes.  The reason being, the query parameter is only available on the initial pageload, and is lost as we navigate between steps.  

Not 100% sure about this; maybe it's better to keep it responsive and to modify use-step-navigation so it keeps the anchor query param around..